### PR TITLE
Add manual checklist testing to the process

### LINKED ISSUE #5496: Stakeholders can reference a manual checklist for functionality to be tested after a contract upgrade release

### PR DESCRIPTION
**Pull request by nambrot**
### Description

The process currently is light on what should be tested after a release on a testnet

### Tested

- [ ] Running the checklist

### Related issues

- Fixes #5496

### CODE REVIEW

**Review by ewilz — APPROVED** [review]
Looks good! Might be good to expand upon manual testing commands in the future (e.g. getting a validator slashed sounds a bit involved)

File: packages/docs/community/release-process/smart-contracts.md [hunk 1]
```diff
@@ -133,6 +143,21 @@ All releases should be evaluated according to the following tests.
 
 All changes since the last release should be covered by unit tests. Unit test coverage should be enforced by automated checks run on every commit.
 
+### Manual Checklist
+
+After a successful release execution on a testnet, the resulting network state should be spot-checked to ensure that no regressions have been caused by the release. Flows to test include:
+
+- Do a cUSD and CELO transfer
```
Comment (ewilz) [inline packages/docs/community/release-process/smart-contracts.md:150]: could have a bash snippet under each bullet point:
```bash
celocli transfer:dollars --from=<addr> --value=<number> --to=<addr>
celocli transfer:celo --from=<addr> --value=<number> --to=<addr>
```

Comment (nambrot) [reply packages/docs/community/release-process/smart-contracts.md:150]: Great idea!

Comment (ewilz) [reply packages/docs/community/release-process/smart-contracts.md:150]: I think we can expand on these later.

**Review by ewilz — APPROVED** [review]

**Pull request merged**